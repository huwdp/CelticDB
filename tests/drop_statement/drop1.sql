CREATE TABLE test;
DROP TABLE test;