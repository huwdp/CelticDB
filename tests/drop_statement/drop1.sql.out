Parser: Create table named 'test'
Parser: Drop table named 'test'
Command: Create statement
Command: Drop statement
